ethcore/src/engines/authority_round/mod.rs

```diff
@@ -924,10 +927,10 @@ mod tests {
 		let addr = tap.insert_account("0".sha3().into(), "0").unwrap();
 		let mut parent_header: Header = Header::default();
 		parent_header.set_seal(vec![encode(&0usize).into_vec()]);
-		parent_header.set_gas_limit(U256::from_str("222222").unwrap());
```

Comment:
`from_str` requires us to import `std::str::FromStr`. `parse` does not and is more idiomatic